src/app/loginComponents/accounts/internal/change-username/change-username.component.html

```diff
@@ -0,0 +1,29 @@
+<p *ngIf="showText">
+  Your current username is <strong>{{user?.username}}</strong>. It may have been auto-generated when you initially created a Dockstore account.
+  You can update your username only if you meet certain conditions.
+</p>
+
+<p>You <strong>can<span *ngIf="!extendedUser?.canChangeUsername">'t</span></strong> change your username.</p>
+
+<div class="change-username-form">
+  <mat-form-field>
+    <input matInput placeholder="Username" name="username" [(value)]="username" (keyup)="checkIfUsernameExists($event)" id="username">
```

Comment:
change from keyup to something with debounceTime, search formInputDebounceTime for example.  Or, put a TODO